ui/desktop/app/routes/scopes/scope/projects/sessions/session.js

```diff
@@ -20,7 +20,9 @@ export default class ScopesScopeProjectsSessionsSessionRoute extends Route {
    * @return {SessionModel}
    */
   async model({ session_id }) {
-    const session = await this.store.findRecord('session', session_id);
+    const session = await this.store.findRecord('session', session_id, {
+      reload: true,
```

Comment:
This technically shouldn't have a forced reload here since it should be using the cached version if it exists due to the [logic](https://github.com/hashicorp/boundary-ui/blob/0487cbc808fa45e78c6d2e854175da4b8fe8c926/ui/desktop/app/controllers/scopes/scope/projects/targets/index.js#L129) we have in case a user doesn't have permissions to read a session.

We'd be making an extra call that would fail otherwise.

Reply:
Ah okay that makes sense, tho the only reason i added it was due to a test failing because a badge on the sessions detail page wasn't correctly displaying text (time remaining). I'll remove this.

Reply:
Hmm I wonder why the cached data doesn't have the proper expiration_time actually...

Reply:
The weird thing is that I did a couple of `console.log`s to look into this and the model showed to have `expiration_time` defined so I am unsure why it's not displaying properly. It's also only an issue in tests 🤔

Reply:
I bet it's something weird happening between the mirage data and when it gets normalized in the client daemon handler since without the reload we're just pulling the data from that when it gets pushed to the store...

Reply:
It's definitely something weird happening...inside the client daemon handler when we normalize the response manually, the issue is happening [here](https://github.com/hashicorp/boundary-ui/blob/cdb73f34876b68d4e4772847e7772882212d5cae/addons/api/addon/serializers/application.js#L246) when we normalize the response, the call to super is setting the fields to null for some reason, it looks like it does it for all date fields 🤔

Reply:
could it be because date fields use the date data type? I wonder if it happens for any fields that are not strings... Thx for pointing out where it happens, ill investigate some more on this.

Reply:
```
{
    "id": "ss_ldyksmlz79",
    "type": "session",
    "attributes": {
        "target_id": "t_agrcgigmr7",
        "status": "active",
        "created_time": null,
        "updated_time": null,
        "expiration_time": null,
        "version": 41236729692160,
        "scope": {
            "scope": {
                "id": "s_id6ozx3wue",
                "type": "org"
            },
            "id": "s_1sheljmjgm",
            "type": "project",            
            "disabled": false,
            "updated_time": "2023-12-15T08:23:25.639Z",
            "created_time": "2023-12-15T09:55:33.176Z",
            "description": "so xerox",
            "name": "commonly memorable whoa",
            "scope_id": "s_1sheljmjgm",
            "isGlobal": false,
            "isOrg": false,
            "isProject": true
        },
        "authorized_actions": [
            "cancel"
        ]
    },
    "relationships": {}
}
```

The scope ones got normalized fine so I'm not sure 😕. Definitely something to investigate

Reply:
Ah I think the issue is I was returning date objects instead of the ISO string